### PR TITLE
Update avatar guidelines to include all staff members

### PR DESCRIPTION
**Pull request by UnsolvedCypher**
It was brought up in #3661 that due to the upcoming redesign of the Teams and Committees page, we should require all Staff and External Staff to upload an avatar that complies with the guidelines that the WQAC created for Delegates. This updates the wording to make the guidelines applicable to all Staff.

### CODE REVIEW

**Comment by AlbertoPdRF** [comment]
Regarding the `any_kind_of_staff?` method that you add, maybe cherry-picking the commit from #3492 where it's defined among others is better, as it's more complete. I did that for #3663, although I still need to work on it a bit before it can be merged.

**Comment by UnsolvedCypher** [comment]
@AlbertoPdRF thanks for the tip, I had no idea about those methods! I've added them to this PR.

File: WcaOnRails/app/models/user.rb [hunk 1]
```diff
@@ -419,6 +433,14 @@ def team_leader?(team)
     self.current_team_members.select { |t| t.team_id == team.id && t.team_leader }.count > 0
   end
 
+  def member_of_any_team?
+    !self.current_team_members.empty?
```
Comment (jonatanklosko) [inline WcaOnRails/app/models/user.rb]: You can avoid the negation with `self.current_team_members.any?`. (Same below)

File: WcaOnRails/app/models/user.rb [hunk 2]
```diff
@@ -431,6 +453,10 @@ def any_kind_of_delegate?
     delegate_status.present?
   end
 
+  def full_delegate?
+    delegate_status == "delegate"
```
Comment (jonatanklosko) [inline WcaOnRails/app/models/user.rb]: Just checking: does this intentionally not include senior delegates?

**Comment by AlbertoPdRF** [comment]
@UnsolvedCypher can you please rebase the PR? Also, please note that most/all of the methods you were adding are already in place (although with different names) and that `External Staff` doesn't exist anymore.

**Comment by AlbertoPdRF** [comment]
Superseded by #4273.

**Pull request closed without merging**